### PR TITLE
note addition

### PR DESCRIPTION
**Pull request by joinimran**
Based on user feedback, I have added a note.

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1222

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/data-loss-prevention-policies.md [hunk 1]
```diff
@@ -85,6 +85,9 @@ If you choose to include or exclude specific SharePoint sites or OneDrive accoun
   
 ### Rules
 
+> [!NOTE]
+> The default behavior of a DLP policy when there is no alert configured is not to alert or trigger. This applies only with default information types. For custom information types system will alert even if there is no action defined in the policy.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-loss-prevention-policies.md]:
```suggestion
> The default behavior of a DLP policy, when there is no alert configured, is not to alert or trigger. This applies only to default information types. For custom information types, the system will alert even if there is no action defined in the policy.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@chrfox - PR has been copy edited and is ready for final review. This is based on issue #1222.

Thank you.

cc: @kenwith

**Pull request merged**